libffi.xcodeproj/project.pbxproj

```diff
@@ -353,18 +353,19 @@
 /* End PBXProject section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		43B5D3FB1D35480D00D1E1FD /* ShellScript */ = {
+		43B5D3FB1D35480D00D1E1FD /* Run Script */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
+			name = "Run Script";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ ! -f \"./compile\" ]\nthen\n./autogen.sh\n/usr/bin/python generate-darwin-source-and-headers.py --only-ios\nfi";
+			shellScript = "if [ ! -f \"./compile\" ]\nthen\nautoreconf -i -f -v\nif [ -f \"../ltmain.sh\" ]\nthen\necho \"fixing ltmain.sh for some reason\"\nmv ../ltmain.sh ./\nautoreconf -i -f -v\nfi\n/usr/bin/python generate-darwin-source-and-headers.py --only-ios\nfi";
```

Comment:
For some (not further investigated) reason the autogen.sh script will misplace the ltmain.sh file when run from within Xcode, so this addition moves that file after the case.